### DISCUSSION TITLE
Signed vs unsigned ints

### DISCUSSION THREAD

**Discussion by Axiverse** [amerkoleci/Vortice.Windows#262] [Polls]
In the DirectX API, a lot of fields are defined as UINTs. For most cases it doesn't really matter if it's signed or unsigned as you won't be reaching the upper limit. However, the convention in .net seems to prefer using ints or longs in almost all cases. In places that don't matter should we lean towards the .net standard or stay true to the API?

**Comment by amerkoleci** [comment]
I would follow the .NET conventions ofc, but for UINT64 I followed the ulong conventions, really not sure which one is the best tho :)
